addons/web/static/tests/views/calendar/calendar_multi_create.test.js

```diff
@@ -14,6 +14,46 @@ import {
 
 import { CalendarModel } from "@web/views/calendar/calendar_model";
 
+class SimpleEvent extends models.Model {
```

Comment:
do you need a new model for your test? I would use one of those already defined if not